tests/test_relay.py

```diff
@@ -0,0 +1,57 @@
+import strawberry
+from strawberry import auto, relay
+from strawberry.schema.config import StrawberryConfig
+
+import strawberry_django
+
+from .models import User
+
+
+def test_relay_with_nodeid():
+    @strawberry_django.type(User)
+    class UserType(relay.Node):
+        id: relay.NodeID[int]
+        name: auto
+
+    @strawberry.type
+    class Query:
+        user: UserType
+
+    schema = strawberry.Schema(query=Query, config=StrawberryConfig(False))
+    str(schema)
+    assert UserType._id_attr == "id"
```

Comment:
Maybe call `UserType.resolve_id_attr()` instead of accessing the private attr?

Reply:
after executing print_schema, the attribute should be set

Reply:
updated